src/dmd/target.d

```diff
@@ -519,9 +520,14 @@ struct Target
      * Returns:
      *      tuple of types if type is passed in one or more registers
      *      empty tuple if type is always passed on the stack
+     *      null if the type is a `void` or argtypes aren't supported by the target
      */
     extern (C++) static TypeTuple toArgTypes(Type t)
     {
+        if (global.params.isSysV64bitABI)
```

Comment:
Stop putting target-specific information in `global.params` please.

Reply:
As a side note, this also has a side effect, but you don't have any parentheses present to denote this.

Reply:
I see. Thank you.